services/order.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
+from django.db import transaction
+from db.models import Ticket, Order, User, MovieSession
+from django.core.exceptions import ValidationError
+
+
+def create_order(tickets: list, username: str, date: str = None) -> Order:
+    with transaction.atomic():
+        user = get_object_or_404(get_user_model(), username=username)
+
+        order = Order.objects.create(user=user)
+
+        if date:
+            try:
+                order.created_at = date
+                order.save()
```

Comment:
Consider validating the date format before assigning it to `order.created_at`. This will help avoid unnecessary exceptions and provide clearer error messages.